# request
float64 target_width
int32 tool_index 0 
bool blocking True
bool depth_compensation False
---
# result
bool result
---